frontend/src/scenes/insights/EditorFilters/samplingFilterLogic.ts

```diff
@@ -40,9 +61,9 @@ export const samplingFilterLogic = kea<samplingFilterLogicType>([
     actions(() => ({
         setSamplingPercentage: (samplingPercentage: number | null) => ({ samplingPercentage }),
     })),
-    reducers(({ values }) => ({
+    reducers(({ props }) => ({
         samplingPercentage: [
-            (values.filters.sampling_factor ? values.filters.sampling_factor * 100 : null) as number | null,
+            defaultSamplingFor(props),
```

Comment:
I guessed the intention was that if filters for an insight changes the sampling percentage should change. There are multiple insightLogic actions that can change the `filters` so I subscribed to the `filters` rather than react to each action... someone changing the insightLogic list of actions that change filters wouldn't know to check this